code/modules/admin/view_variables/topic_basic.dm

```diff
@@ -59,7 +59,7 @@
 		names += componentsubtypes
 		names += "---Elements---"
 		names += sortList(subtypesof(/datum/element), /proc/cmp_typepaths_asc)
-		var/result = input(usr, "Choose a component/element to add","better know what ur fuckin doin pal") as null|anything in names
+		var/result = var/result = tgui_input_list(usr, "Choose a component/element to add", "Add Component", names)
```

Comment:
var/result = var/result

Reply:
It's true though, var/result DOES IN FACT EQUAL var/result